src/collection/collection.service.ts

```diff
@@ -1514,4 +1521,89 @@ export class CollectionService {
 			});
 		}
 	}
+
+	static async fetchContributorsByCollectionId(assignmentId: string) {
```

Comment:
Add a return type to service functions. Something like:
```suggestion
	static async fetchContributorsByCollectionId(assignmentId: string): Promise<ContributorInfo[]> {
```